capstone-app/src/App.js

```diff
@@ -1,26 +1,199 @@
+
+// @material-ui/icons
+
+// core components
+import Header from "components/Header/Header.js";
+import Footer from "components/Footer/Footer.js";
+import GridContainer from "components/Grid/GridContainer.js";
+import GridItem from "components/Grid/GridItem.js";
+import Button from "components/CustomButtons/Button.js";
+import HeaderLinks from "components/Header/HeaderLinks.js";
+import Parallax from "components/Parallax/Parallax.js";
+
+import styles from "assets/jss/material-kit-react/views/landingPage.js";
+
+// Sections for this page
+import ProductSection from "./Sections/ProductSection.js";
+// import TeamSection from "./Sections/TeamSection.js";
+import WorkSection from "./Sections/WorkSection.js";
+
+const dashboardRoutes = [];
+
+const useStyles = makeStyles(styles);
+
+export default function LandingPage(props) {
+  const classes = useStyles();
+  const { ...rest } = props;
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div>
+      <Header
+        color="transparent"
+        routes={dashboardRoutes}
+        brand="Material Kit React"
+        rightLinks={<HeaderLinks />}
+        fixed
+        changeColorOnScroll={{
+          height: 400,
+          color: "white"
+        }}
+        {...rest}
+      />
+      <Parallax filter image={require("assets/img/landing-bg.jpg")}>
+        <div className={classes.container}>
+          <GridContainer>
+            <GridItem xs={12} sm={12} md={6}>
+              <h1 className={classes.title}>Your Story Starts With Us.</h1>
+              <h4>
+                Every landing page needs a small description after the big bold
+                title, that{"'"}s why we added this text here. Add here all the
+                information that can make you or your product create the first
+                impression.
+              </h4>
+              <br />
+              <Button
+                color="danger"
+                size="lg"
+                href="https://www.youtube.com/watch?v=dQw4w9WgXcQ&ref=creativetim"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                <i className="fas fa-play" />
+                Watch video
+              </Button>
+            </GridItem>
+          </GridContainer>
+        </div>
+      </Parallax>
+      <div className={classNames(classes.main, classes.mainRaised)}>
+        <div className={classes.container}>
+          <ProductSection />
+          
+          <WorkSection />
+        </div>
+      </div>
+      <Footer />
     </div>
   );
 }
 
-export default App;
+
```

Comment:
we can reference old code by checking out older commits, so it doesn't need to stay here